src/build.py

```diff
@@ -85,7 +85,7 @@ def main():
   print 'Applying minified libs'
   for f in glob.glob('build/lib/*.js'):
     os.remove(f)
-  for f in glob.glob('build/lib/min/*.js'):
+  for f in glob.glob('build/lib/min/*'):
```

Comment:
the wasm file has to be in the same folder as sqlite3.js so we want to import all files from the `min` folder during the build